src/lua/utils.h

```diff
@@ -258,6 +258,19 @@ void
 luaL_register_module(struct lua_State *L, const char *modname,
 		     const struct luaL_Reg *methods);
 
+/**
+ * @brief Push a value with specified key into the table at the top
+ * of the Lua stack.
+ * @param name key for the value.
+ * @param method method that pushes the value on the Lua stack.
+ * @param value value.
+ */
+#define luaL_tablepush(name, method, value) do	{	\
```

Comment:
I don't like this helper, because it doesn't work if:
 - Lua state has other name than `L`. E.g. we use `tarantool_L` sometimes.
 - It doesn't work if the table isn't on top of the Lua stack.
 - It doesn't work if the method takes more than one argument.

Since it's a trivial macro, I think that you should simply redefine it in your source file instead.